### PR TITLE
fix #79180 schoener-wohnen.de

### LINKED ISSUE #79180: www.schoener-wohnen.de
### Issue URL (Annoyance)
 [https://www.schoener-wohnen.de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.schoener-wohnen.de%2F)
### Comment
Username: @andy-kl
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 beta 1
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2), <br>Disable cache for third-party requests, <br>Block Push API, <br>Block Location API, <br>Block Flash, <br>Block Java, <br>Hide your Referrer from third-parties (217.237.150.188)
DNS filtering: | server: `https://dns.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> Fanboy's Anti-Facebook List, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> Fanboy's Social Blocking List, <br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> Latvian List, <br/>Icelandic ABP List, <br/>Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>Polish Anti Adblock Filters, <br/>Dandelion Sprout's Nordic Filters, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Spanish/Portuguese
Userscripts: | https://userscripts.adtidy.org/beta/assistant/4.3/assistant.user.js,<br>https://userscripts.adtidy.org/beta/popup-blocker/2.5/popupblocker.user.js,<br>https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by wout-codes**
#79180

### CODE REVIEW

**Review by BlazDT — CHANGES_REQUESTED** [review]
Breaks video on https://www.schoener-wohnen.de/moebel/40875-vdo-designer-markus-jehs-und-juergen-laub-stellen-ihr-sofa-moss-vor

Seems to need to accept cookie.

**Comment by wout-codes** [comment]
Something like this should work: 

```
schoener-wohnen.de#%#AG_onLoad(function() {if (!document.cookie.includes("euconsent-v2")) {var g = new MutationObserver(function() {var b = document.querySelector('button[aria-label="Zustimmen"]');var c = document.querySelector('html');b && b.click();c && c.style.setProperty('overflow', 'static', 'important');c && c.style.setProperty('position', 'auto', 'important');});g.observe(document, {childList: !0,subtree: !0});setTimeout(function() {g.disconnect()}, 1E4)}});
```

**Comment by wout-codes** [comment]
Added a rule which should do the work.

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -58,6 +58,8 @@ semprotyuk.com##div[style^="margin-bottom:150px;"][style*="padding-right:2em;"]
 ||za.zalo.me/v*/brp?fts=
 ||za.zalo.me/v*/w/_zaf.gif
 ||api.eigene.io/rec/
+||guj.de/js/gujTracker.js
+/tracking.js$domain=schoener-wohnen.de
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:62]: I don't see these requests. Do you still see them?

Comment (wout-codes) [reply SpywareFilter/sections/specific.txt:62]: @Alex-302 Yes.
image

Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:62]: Ok.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**